src/plugins/workspace/server/saved_objects/integration_tests/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -59,6 +61,8 @@ describe('WorkspaceSavedObjectsClientWrapper', () => {
   let osd: TestOpenSearchDashboardsUtils;
   let permittedSavedObjectedClient: SavedObjectsClientContract;
   let notPermittedSavedObjectedClient: SavedObjectsClientContract;
+  let groupIsDashboardAdminSavedObjectedClient: SavedObjectsClientContract;
+  let UserIdIsdashboardAdminSavedObjectedClient: SavedObjectsClientContract;
```

Comment:
Prefer `userIdIsdashboardAdminSavedObjectedClient`

Reply:
There is no relevant logic of users/groups in the wrapper, so I uniformly use the `dashboardAdminSavedObjectedClient` name in the test file.